pyccel/codegen/printing/ccode.py

```diff
@@ -1438,6 +1438,28 @@ def _print_NumpyRandint(self, expr):
     def _print_NumpyMod(self, expr):
         return self._print(PyccelMod(*expr.args))
 
+    def _print_NumpySum(self, expr):
+        '''
+        Convert a call to numpy.sum to the equivalent function in C.
+        '''
+        if not isinstance(expr.arg, (NumpyArray, Variable, IndexedElement)):
+            raise TypeError(f'Expecting a NumpyArray, given {type(expr.arg)}')
+        dtype, prec, name = (expr.arg.dtype,
+                             expr.arg.precision,
+                             self._print(expr.arg))
+        if prec == -1:
+            prec = 8
```

Comment:
You should use the `default_precision` from `ast.datatypes` not define your own precision. The current method will probably lead to failures for windows with integers as you will see when you add tests ;)
```suggestion
            prec = default_precision[self._print(dtype)]
```